hr_employee_service_length/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=======================
+Employee Service Length
+=======================
+
+Calculates service length of employee
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/hr
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "Employee Service Length" in your addons
+* install the module
+
+Configuration
+=============
+
+None Needed
+
+Usage
+=====
+
```

Comment:
as @adrienpeiffer says, take a look to the template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst and fill the  `Usage` section with the `Try me on runbot` link.